src/rewritable_units/tokens/end_tag.rs

```diff
@@ -49,20 +54,20 @@ impl<'i> EndTag<'i> {
     #[inline]
     #[doc(hidden)]
     #[deprecated(note = "use set_name_str")]
-    pub fn set_name(&mut self, name: Bytes<'static>) {
+    pub fn set_name(&mut self, name: BytesCow<'static>) {
```

Comment:
Was this meant to be `pub(crate)`?

Reply:
This is kept for semver-compat. It should have taken `&str` instead of a type that cannot be named and exposes internals.